flow-typed/redux/transactions.js

```diff
@@ -1,10 +1,12 @@
 // @flow
 
 declare type Hash = string
+// TransactionId consist of: `${txHash}${?txLogIndex}`
 declare type TransactionId = string
 
 declare type TransactionFromBlockExplorer = {|
   +hash: Hash,
+  +nonce: string,
```

Comment:
To avoid parsing of string I suggest to make it as `number` type.
Type of that param in response is number

Reply:
But value from node is `string`